File: 大一上/图灵班/程序设计荣誉课程-1/yoj全解/YOJ-1/yoj-1.cpp
/*
 * @Author: zmlad
 * @Date: 2025-11-27 18:21:45
 */
/*
 * 程序目的：输出 "hello world" 字符串。
 * 实现方式：
 * 1. 包含输入输出流头文件 iostream，以使用 cout 对象进行输出。
 * 2. 使用命名空间 std，避免重复书写 std:: 前缀。
 * 3. 定义主函数 main，作为程序入口点。
 * 4. 使用 cout 和流插入运算符 << 输出字符串 "hello world"，并添加 endl 以换行并刷新输出缓冲区。
 * 5. 返回 0 表示程序正常退出。
 */
#include <iostream>
using namespace std;

int main() {
    cout << "hello world" << endl;
    return 0;
}

File: 大一上/图灵班/程序设计荣誉课程-1/yoj全解/YOJ-1/yoj-1.py
'''
Author: zmlad
Date: 2025-11-27 18:21:59
'''
"""
程序目的：输出 "hello world" 字符串。
实现方式：
1. 直接使用内置函数 print 输出字符串，Python 的 print 函数默认会自动添加换行。
2. 虽然 Python 不需要显式定义主函数，但代码写在 if __name__ == "__main__": 块中可以确保模块作为脚本运行时才执行，提高代码模块化。
"""
if __name__ == "__main__":
    print("hello world")